src/iitkgp_erp_login/erp.py

```diff
@@ -189,6 +189,9 @@ def login(
         sessionToken=sessionToken
     )
 
+    # Request for OTP
+    request_otp(headers=headers, session=session, login_details=login_details, log=LOGGING)
+
```

Comment:
There was a bug, due to which this call was merged into `getOTP()` function call.